orca-retrofit/orca-retrofit.gradle

```diff
@@ -37,6 +37,8 @@ dependencies {
   testImplementation("org.junit.jupiter:junit-jupiter-params")
   testImplementation("org.mockito:mockito-core")
   testImplementation("org.springframework.boot:spring-boot-test")
+  testImplementation("com.squareup.okhttp3:mockwebserver")
+  implementation("com.squareup.retrofit2:converter-jackson")
```

Comment:
```suggestion
  testImplementation("com.squareup.retrofit2:converter-jackson")
```
?

Reply:
can't believe I missed that, I'll update it